Error: WebGL unsupported in this browser, use "pixi.js-legacy" for fallback canvas2d support.

```
main-6bef3842d6b2422a0213.js:1 Error: WebGL unsupported in this browser, use "pixi.js-legacy" for fallback canvas2d support.
    at Function.e.create (6.b87f350c84c07372dab0.js:1)
    at 6.b87f350c84c07372dab0.js:1
    at new t (6.b87f350c84c07372dab0.js:1)
    at e.value (6.b87f350c84c07372dab0.js:1)
    at commons.120a8eedac9ac3a95b46.js:1
    at t.unstable_runWithPriority (commons.120a8eedac9ac3a95b46.js:1)
    at Yo (commons.120a8eedac9ac3a95b46.js:1)
    at Ml (commons.120a8eedac9ac3a95b46.js:1)
    at gl (commons.120a8eedac9ac3a95b46.js:1)
    at commons.120a8eedac9ac3a95b46.js:1
```

--- a/components/ImageCanvas.tsx
+++ b/components/ImageCanvas.tsx
@@ -9,6 +9,7 @@ import PixiComponentText from './PixiComponentText'
 import { ICanvasState } from '../logic/reducer'
 import { RGBColor } from 'react-color'
 import { IFontInfo } from '../logic/reducerFont'
+import { TextStyleFontWeight } from 'pixi.js-legacy'
 
 interface ITextCanvasProps extends ICanvasState {
   fontInfo: IFontInfo
@@ -91,7 +92,7 @@ const ImageCanvas: FunctionComponent<ITextCanvasProps> = props => {
         style={{
           // align: 'left',
           fontSize,
-          fontWeight: String(fontWeight),
+          fontWeight: String(fontWeight) as TextStyleFontWeight,
           lineHeight,
           // fill: ['#ffffff', '#00ff99'], // gradient
           fill: [colorTextAsNumber],

--- a/components/PixiComponentText.tsx
+++ b/components/PixiComponentText.tsx
@@ -2,6 +2,7 @@ import { PixiComponent } from '@inlet/react-pixi'
 import { TextStyle, Text as PIXIText} from 'pixi.js'
 import { throttle } from 'lodash'
 import { IFontInfo, getFontFamilyFromFontInfo } from '../logic/reducerFont'
+import { TextStyleFontWeight } from 'pixi.js-legacy'
 
 type TPropsPixiComponentText = {
   text: string
@@ -12,7 +13,7 @@ type TPropsPixiComponentText = {
   style: {
     // align: 'left',
     fontSize: number
-    fontWeight: string
+    fontWeight: TextStyleFontWeight
     lineHeight: number
     // fill: ['#ffffff', '#00ff99'], // gradient
     fill: string | number | string[] | number[] | CanvasGradient | CanvasPattern

--- a/package.json
+++ b/package.json
@@ -15,7 +15,7 @@
     "lodash": "^4.17.15",
     "next": "9.1.1",
     "nice-color-palettes": "^3.0.0",
-    "pixi.js": "^5.1.5",
+    "pixi.js-legacy": "^6.1.3",
     "react": "16.11.0",
     "react-color": "^2.17.3",
     "react-dom": "16.11.0",

--- a/package.json
+++ b/package.json
@@ -34,6 +34,6 @@
     "@types/trianglify": "^1.2.0",
     "@types/webfontloader": "^1.6.29",
     "now": "^16.4.4",
-    "typescript": "^3.6.4"
+    "typescript": "^4.4.3"
   }
 }

